### ISSUE TITLE
openflow-bandwidth with MININET

### ISSUE THREAD

**Issue by akouSDN** [biirdy/openflow_bandwidth#1]
Dear @lyndon160 @biirdy , pealse can you put a tutorial on how to use openflow-bandwidth with MININET topology.

regards,

**Comment by biirdy** [comment]
@akouSDN , we would like to make a tutorial for Mininet, although OvS lacks the metering support required for the bandwidth enforcing. We were using this controller with HP hardware switches. 
There are alternative software switches that support metering that could be installed, and along with the possibility of metering being implemented in OvS, a tutorial could be made in the future.

Regards,
Jamie

**Comment by akouSDN** [comment]
@biirdy Ok thanks for replay. CPqD switch support mete tables. So, its suitable choice CPqD with your BW app and mininet. Waiting for tutorial. Please when you finish it just let me know.

Regards

**Comment by lyndon160** [comment]
@akouSDN yes that's right. CPqD does support meters and will work with this project; I've tried it already. It doesn't have the best performance and is buggy requiring some workarounds to get it to work but maybe be suitable for an experiment. We will look into making a tutorial for mininet during the next week,  though it shouldn't be difficult to setup yourself. All you need to do is start mininet with CPqD switch and point ryu to that switch. The rest is already documented. 

What are you trying to use this for?

**Comment by akouSDN** [comment]
Ok many thanks @lyndon160 waiting for tuto. Furthermore, how many switch can be supported by your BW app !

Basically, I want to improve the  QoS for multimedia applications using SDN. If you have some time add me is Skype 'akoubba1' we can discuses.
